integration-test/README.md

```diff
@@ -163,3 +163,19 @@ Selected [promotion test cases](https://github.infra.cloudera.com/CDH/cdpmc-qe/t
             - `AZURE_CLIENT_SECRET`: application password
             - `AZURE_TENANT_ID`: tenant ID which is associated with the selected application
 - Run the predefined test suites for L0 promotion. These suite files can be found at [src/main/resources/testsuites/e2e/l0promotion](src/main/resources/testsuites/e2e/l0promotion) directory. Tests can be [run in IDEA](#run-integration-tests-in-idea) or [in Terminal](#run-a-selected-e2e-test-in-terminal).
+
+## Know-how
+
+### PKIX SSL Error
+If you can not run integration tests locally because of `PKIX SSL error` at `Mock Image Catalog call`, then you can solve it with:
+```
+sudo keytool -import -alias mock-infra -noprompt -file "/home/peter/prj/cloudbreak/mock-infrastructure/src/main/resources/keystore/infrastructure-mock.cer" -keystore /etc/ssl/certs/java/cacerts -storepass changeit
```

Comment:
Please use ${USER} placeholder instead of peter or just prefix with ~/

Beyond the linux command, we can add the OSX one as well:

`keytool -import -alias mock-infra -noprompt -file ~/prj/hortonworks/cloudbreak/mock-infrastructure/src/main/resources/keystore/infrastructure-mock.cer -keystore /opt/homebrew/opt/openjdk@11/libexec/openjdk.jdk/Contents/Home/lib/security/cacerts -storepass changeit`